src/towncrier/newsfragments/591.bugfix.rst

```diff
@@ -0,0 +1 @@
+Leveraged pathlib in most file operations.
```

Comment:
It this a bugfix or just a .misc internal cleanup ?

Reply:
Good call. I'd forgotten there might be other options than the defaults. Agreed it's misc.

Reply:
Renamed in [416a2c8](https://github.com/twisted/towncrier/pull/591/commits/416a2c8fffa643a6988e3df91b97b61848045cfd).